packages/Utils/test/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Miscellaneous_test
+  SOURCES tstMiscellaneous.cpp ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
```

Comment:
If you only specify `serial` it will not be compiled for MPI builds. Not sure why it won't work if you remove `COMM` altogether but maybe the default behavior is serial then